pkg/volume/vspherevolume/attacher_test.go

```diff
@@ -241,8 +241,7 @@ func (testcase *testcase) AttachDisk(diskName string, storagePolicyName string,
 	if expected.diskName == "" && expected.nodeName == "" {
 		// testcase.attach looks uninitialized, test did not expect to call
 		// AttachDisk
-		testcase.t.Errorf("Unexpected AttachDisk call!")
-		return "", errors.New("Unexpected AttachDisk call!")
+		return "", errors.New("Unexpected AttachDisk call")
```

Comment:
why the call to testcase.t.Errorf("Unexpected AttachDisk call!") was removed? This is linter change, I'd suggest to change functionality separately

Reply:
I dont remember changing this but ill update it, probably came via rebase

Reply:
Will update